### PR TITLE
User-defined colors & deprecated function

### PR DESCRIPTION
**Pull request by tehhowch**
- Swaps a few more colors from the map drawing system to be user-defined
 - Removes an unused function declaration (function removed in https://github.com/endless-sky/endless-sky/commit/cdb1cb881d04b246feee03dff4ec0028f34f6327#diff-e0a59618d76b767cb62fa83a9f007c59L295)

### CODE REVIEW

**Comment by endless-sky** [comment]
I think it'd just be better to use the "medium" and "dim" colors in place of "near" and "far." Note that when drawn on a background with a grey level of about 0.15 (32 to 48 in grey levels), the "medium" color (0.5 additive; 0.15 + 0.50 = 0.65) will look nearly the same as the "close" color (1 at 0.6 alpha; 0.15 * 0.4 + 1 * 0.6 = 0.66). Similarly, the "dim" color (0.25 additive; 0.15 + 0.25 = 0.400) has nearly the same effect as the "far" color (1 at 0.3 alpha; 0.15 * 0.7 + 1 * 0.3 = 0.405).

**Comment by endless-sky** [comment]
(The map panel is one of the earliest parts of the game; I probably created it before the named colors in the interface even existed. That's why it's sometimes inconsistent about using those colors.)

**Comment by tehhowch** [comment]
Added a button to visually convey that the `Find` dialog exists and has a hotkey (Refs #1893, #1030, perhaps more).

File: data/interfaces.txt [hunk 1]
```diff
@@ -825,6 +824,12 @@ interface "map buttons" bottom right
 	button i "M_issions"
 		from -170 -50 to -80 0
 	
+	active
+	sprite "ui/dialog cancel"
+		from -170 -90 to -80 -40
+	button f "_Find..."
+		from -170 -90 to -80 -40
```
Comment (endless-sky) [inline data/interfaces.txt]: Having so many buttons in the bottom right corner that they have to be arranged in a grid two rows high is rather awful UI design. The +/- buttons are already bad enough, but adding this makes the UI look even more cluttered.

Comment (tehhowch) [reply data/interfaces.txt]: I wanted to go for the lower left corner, but it overlaps the pointer panel when viewing missions.
My second choice was above the +/- buttons, but we'd need to add a rotation keyword to the interface sprite specification (or add a a rotated version of ui/dialog cancel sprite, which seemed like too much) to get the sprite decorations between the +/- and the dialog sprite to play nicely.

**Pull request merged**